app/questions.json

```
@@ -136,4 +136,10 @@
   "type": 0,
   "text": "Spoink is a bouncy psychic-type Pokémon that should not be confused with Splunk, a software suite for analyzing log data.",
   "img": "spoink.png"
+}, {
+  "name": "Debezium",
+  "url": "https://debezium.io",
+  "type": 0,
```

```suggestion
  "type": 1,
```

Type is `1` when it's BigData, `0` when a Pokemon.